sandbox.go

```diff
@@ -722,6 +722,13 @@ func (sb *sandbox) restoreOslSandbox() error {
 		if len(i.llAddrs) != 0 {
 			ifaceOptions = append(ifaceOptions, sb.osSbox.InterfaceOptions().LinkLocalAddresses(i.llAddrs))
 		}
+		if len(ep.virtualIP) != 0 {
+			_, secVIP, err := net.ParseCIDR(fmt.Sprintf("%s/32", ep.virtualIP.String()))
```

Comment:
Please replace the if block with
`secIP := &net.IPNet{IP: ep.VirtualIP, Mask: net.CIDRMask(32,32))}`